### PR TITLE
Examples to showcase HyperSDK Features

### PR DESCRIPTION
**Pull request by x-du**
I added a few examples to showcase HyperSDK features. 

- 1. Guest Authentication. 
- 2. More complete authentication workflow. 
- 3. Select Cards
- 4. Disable and enable cards on the fly
- 5. Search API

For above examples, I added jsfiddle button, so people can modify the code on jsfiddle without downloading the code to local machine. 

In addition, I also change the color theme to match MicroStrategy's color and changed the title of the documentation page to MicroStrategy Hyper SDK  

Please review them, merge to the public github, or merge it to microstrategy's github then merge to public. 

You can view the demos live at https://x-du.github.io/hyper-sdk/

### CODE REVIEW

File: feature_showcase/style/jsfiddle.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.jsfiddle-button {
```
Comment (lucianchen) [inline feature_showcase/style/jsfiddle.css:1]: @x-du could we apply similar styling as the code examples to be consistent?

Comment (liangchenmstr) [inline feature_showcase/style/jsfiddle.css:1]: @x-du could we apply similar styling as the code examples to be consistent?

**Comment by liangchenmstr** [comment]
> I added a few examples to showcase HyperSDK features.
> 
> * 1. Guest Authentication.
> * 1. More complete authentication workflow.
> * 1. Select Cards
> * 1. Disable and enable cards on the fly
> * 1. Search API
> 
> For above examples, I added jsfiddle button, so people can modify the code on jsfiddle without downloading the code to local machine.
> 
> In addition, I also change the color theme to match MicroStrategy's color and changed the title of the documentation page to MicroStrategy Hyper SDK
> 
> Please review them, merge to the public github, or merge it to microstrategy's github then merge to public.
> 
> You can view the demos live at https://x-du.github.io/hyper-sdk/

**Comment by liangchenmstr** [comment]
> I added a few examples to showcase HyperSDK features.
> 
> * 1. Guest Authentication.
> * 1. More complete authentication workflow.
> * 1. Select Cards
> * 1. Disable and enable cards on the fly
> * 1. Search API
> 
> For above examples, I added jsfiddle button, so people can modify the code on jsfiddle without downloading the code to local machine.
> 
> In addition, I also change the color theme to match MicroStrategy's color and changed the title of the documentation page to MicroStrategy Hyper SDK
> 
> Please review them, merge to the public github, or merge it to microstrategy's github then merge to public.
> 
> You can view the demos live at https://x-du.github.io/hyper-sdk/

Hi @x-du the examples overall looks great, thanks for the add-ons to the documentation! I think after the styling changes for the example pages, it should be good to be merged.

**Pull request merged**